tutorials/02_installation.md

```diff
@@ -36,26 +36,41 @@ sudo apt-get install libignition-physics3-dev
 
 **Build from source**
 
-Run the following to install dependencies
-```{.sh}
-sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
-sudo apt-get build-dep -y ignition-physics3
-```
+1. Run the following to install dependencies
 
-Clone the ign-physics repository from GitHub
-```{.sh}
-git clone https://github.com/ignitionrobotics/ign-physics -b master
-```
+    ```{.sh}
+    sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
+    sudo apt-get build-dep -y ignition-physics3
```

Comment:
This won't work yet, because `ignition-physics3` hasn't been released.

We could instruct users to get dependencies from these `packages` files once #94 is merged:

https://github.com/ignitionrobotics/ign-physics/tree/ign-physics2/.github/ci